src/opnsense/www/js/opnsense_theme.js

```diff
@@ -30,176 +30,180 @@
 /* hook-in additional theme functionality */
 
 $(document).ready(function () {
-    // traverse loaded css files
-    var toggle_sidebar_loaded = false;
-    var $window = $(window);
-    var winheight = $(window).height();
-    var nomouse = "mouseenter mouseleave";
-    var layer1_a = $('#mainmenu > div > a');
-    var layer1_div = $('#mainmenu > div > div');
-    var layer2_a = $('#mainmenu > div > div > a');
-    var layer2_div = $('#mainmenu > div > div > div');
-    var navigation = $("#navigation");
-    var events = {
-        mouseenter: function () {
-            $("#navigation.col-sidebar-left").css("width", "415px");
-            var that = $(this);
-            if (that.next("div").hasClass("in")) {
-                /* no action needed */
-            } else if ((that.next().is("a")) || (that.is("a:last-child"))) {
-                activeremove(this);
-            } else {
-                var divtop = that.offset().top - $window.scrollTop();
-                var divheight = that.next("div").height();
-                var currentheight = (divtop + divheight);
-                that.trigger("click");
-                if (currentheight > winheight) {
-                    that.next("div").css("margin-top", -divheight - (that.is(layer1_a) ? 3 : 0));
-                }
+  // traverse loaded css files
```

Comment:
indent changed from 4 to 2 spaces, maybe you can adjust that in your IDE?

Reply:
I actually thought it would be better because you used 2 instead of 4 characters. to undo the whole thing is really difficult. would have to insert everything new. is there a simple possibility? regards rené